src/app/components/Hero.js

```diff
@@ -10,6 +10,25 @@ export default class Hero extends Component {
     background: PropTypes.string.isRequired,
   };
 
+static defaultProps = {
+    pageTitleType: "type-hero",
```

Comment:
Add `pageTitleType: PropTypes.oneOf(['type-hero'])` to `propTypes`.

Also add `className: PropTypes.string` to `propTypes`.

Add `className: ''` to default props.